src/Parser/OrientationParser.js

```diff
@@ -0,0 +1,111 @@
+import * as THREE from 'three';
+
+const DEG2RAD = THREE.Math.DEG2RAD;
+
+// The transform from world to local is  RotationZ(heading).RotationX(pitch).RotationY(roll)
+// The transform from local to world is (RotationZ(heading).RotationX(pitch).RotationY(roll)).transpose()
+THREE.Quaternion.prototype.setFromRollPitchHeading = function setFromRollPitchHeading(roll = 0, pitch = 0, heading = 0) {
```

Comment:
I'm sorry for being always like this, but I don't like changing the prototype of external things of itowns. Why not make those methods static and add a `target` parameter to them to replace the `this` ?

Reply:
Another option is to have a CRS-aware subclass of THREE.Quaternion (itowns.Quaternion ?) with these added methods (kinda like itowns.Coordinate is a CRS-aware version of THREE.Vector3).

Reply:
I'd prefer that yes, maybe we could give it a try.

Reply:
ok, I've changed the code to use target parameter.
@mbredif No need to add another itownsQuaternion object since this code is the parser responsability.